pkg/networkservice/common/excludedprefixes/utils.go

```diff
@@ -42,31 +42,44 @@ func removeDuplicates(elements []string) []string {
 	return result
 }
 
-func watchFile(ctx context.Context, path string, onChanged func([]byte)) error {
+func watchFile(ctx context.Context, filePath string, onChanged func([]byte)) error {
 	watcher, err := fsnotify.NewWatcher()
 	if err != nil {
 		return err
 	}
-	if err := watcher.Add(path); err != nil {
+
+	directoryPath, fileName := filepath.Split(filePath)
```

Comment:
Does this work when `directoryPath ='"`?

Reply:
Unit test added